symengine/codegen.h

```diff
@@ -355,13 +355,13 @@ class JSCodePrinter : public BaseVisitor<JSCodePrinter, CodePrinter>
     }
 };
 
-std::string ccode(const Basic &x)
+std::string inline ccode(const Basic &x)
```

Comment:
Implementations in header files should be marked inline. Otherwise we can't link two translation units that included this header together.

Reply:
I see, yes.